staging/src/k8s.io/kubectl/pkg/util/templates/templates.go

```diff
@@ -52,12 +52,6 @@ const (
 {{ if $visibleFlags.HasFlags}}{{trimRight (flagsUsages $visibleFlags)}}{{end}}{{ if $explicitlyExposedFlags.HasFlags}}{{ if $visibleFlags.HasFlags}}
 {{end}}{{trimRight (flagsUsages $explicitlyExposedFlags)}}{{end}}
 
-{{end}}`
```

Comment:
Leave these as is, this way when you apply the previous suggestion and run just `kubectl` you'll get nice info what `kubectl` is, w/o it you won't. Compare yours output with:
```
$ kubectl
kubectl controls the Kubernetes cluster manager.

 Find more information at: https://kubernetes.io/docs/reference/kubectl/overview/
...
```
yours is missing above

Reply:
done